R/retentionIndexing.R

```diff
@@ -0,0 +1,54 @@
+#' @title Convert retention times to retention indices
+#'
+#' @description
+#'
+#' `rtimeIndexing` uses a list of known substances to convert retention times to
+#'     retention indices. By default linear interpolation is performed.
+#'
+#' @param x `numeric` vector with retention times
+#' 
+#' @param y `data.frame`data.frame containing two columns, where the first 
+#'     holds the retention times of the indexing substancs and the second the
+#'    actual index value
+#' 
+#' @param FUN `function` function defining how the conversion is performed, 
+#'     default is linear interpolation
+#'     
+#' @param ... additional parameter used by `FUN`
+#'
+#' @return `numeric` vector of same length as x with retention indices
+#'
+#' @author Michael Witting
+#'
+#' @export
+#'
+#' @examples
+#'
+#' rti <- data.frame(rtime = c(1,2,3),
+#' rindex = c(100,200,300))
+#' rtime <- c(1.5, 2.5)
+#' rtimeIndexing(rtime, rti)
+#' 
+rtimeIndexing <- function(x,
```

Comment:
I'm wondering: would it not be better to call this function `rtimeIndex` or `indexRtime` to make it explicit that you want to actually do the indexing. I would call the whole process rtime indexing, but IMHO the function should be called without the *ing*. Thus, to make it clear that you want to *index the retention times* I would favout `indexRtime` over `rtimeIndex` (which would also mean *return the rtime index*)

Reply:
Done.